app/models/setting.rb

```diff
@@ -83,4 +88,16 @@ def self.invoice_reminder_in_days
   def self.wishlist_size
     Setting.find_by(code: :wishlist_size).value.to_i
   end
+
+  class << self
+    private
+
+    def json_hash?(text)
+      JSON.parse(text).is_a?(Hash)
+    rescue JSON::ParserError
```

Comment:
It's quite dangerous to silently discard an error. Someone (maybe you) will have hard time figuring our why some code doesn't work due to this.

Reply:
I'll modify this part as for #387 .